person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_accessor :name, :age
+  attr_writer :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @age = age
```

Comment:
* Kindly create an instance variable `@id` for the class `Person` as stated in the requirement [Refer here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Instance%20vars%3A%20%40id%2C%20%40name%2C%20and%20%40age.)